docs/source/management_commands.rst

```diff
@@ -19,6 +19,8 @@ For example, it binds to address of 10.0.0.1:21::
 
    $ python manage.py ftpserver 10.0.0.1:21
 
+Keep in mind that you need root permissions if you use well known ports (0 - 1024) for ``ftpserver``.
```

Comment:
Well known ports are 0 to 1023.
refs: https://datatracker.ietf.org/doc/html/rfc6056#section-2.1
Please update.

Reply:
Is updated